### DISCUSSION TITLE
Auto Copy confirmation code to the system clipboard

### DISCUSSION THREAD

**Discussion by simanga-dev** [dunst-project/dunst#1265] [Ideas]
Is there a way or a script that I can add so that if an email notification has a confirmation code, it automatically gets copied to my system clipboard?

**Comment by bynect** [comment]
if the confirmation code is in the body of the notification you can attach a script that scans the body with a regular expression and adds it to your clipboard

**Reply by simanga-dev** [reply]
Thank you for your response. How do I do that?

**Reply by bynect** [reply]
This is similar to your use case #753
